client/signup/config/flows-pure.js

```diff
@@ -444,7 +444,10 @@ export function generateFlows( {
 			lastModified: '2024-05-16',
 			enableBranchSteps: true,
 			hideProgressIndicator: true,
-			enablePresales: false,
+			enableHelpCenter: true,
+			get helpCenterButtonText() {
+				return translate( 'Questions? Contact our site building team' );
```

Comment:
Strings are not translated yet, but we can merge since this feature is behind a flag.

Reply:
Just out of curiosity, why isn't this getter just a prop? Translations are loaded a bit later?

Reply:
Yes, translations are loaded from the server and won't be present when the function is called. #62101 has more details.